health/checks/checks_test.go

```diff
@@ -23,3 +29,30 @@ func TestHTTPChecker(t *testing.T) {
 		t.Errorf("Google at Portugal was expected as exists, error:%v", err)
 	}
 }
+
+func TestIPChecker(t *testing.T) {
+	if err := PingChecker("8.8.8.8", maxRTT).Check(); err != nil {
+		t.Errorf("8.8.8.8 was expected as exists")
+	}
+	if err := PingChecker("254.254.254.254", maxRTT).Check(); err == nil {
+		t.Errorf("254.254.254.254 was expected as not exists")
+	}
+	if err := PingChecker("8.8.8.8", impossibleRTT).Check(); err == nil {
+		t.Errorf("8.8.8.8 was expected as exists but not within 1msec")
+	}
+
+}
+
+func TestIPCheckerIPv6(t *testing.T) {
+	t.Skip("Support for IPv6 requires it to be enabled")
```

Comment:
For the IPv6 tests I wrote in `docker/docker` I set up a little test using [go-check](https://github.com/go-check/check) to enable the tests only on hosts when IPv6 is enabled. I included the tests here as they pass on my dual stack machine, but fail on CircleCI. I guess we could address this later...